packages/providers/inpage-providers-hub/src/floatingButton/index.tsx

```
@@ -0,0 +1,746 @@
+                alignItems: 'center',
+                gap: '2px',
+              }}
+            >
+              {securityInfo?.dapp?.origins.map((item) => (
+                <img
+                  src={item.logo}
+                  style={{
+                    width: '16px',
+                    height: '16px',
+                  }}
+                />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)